system-addon/content-src/components/TopSites/TopSitesEdit.jsx

```diff
@@ -133,8 +121,8 @@ export class _TopSitesEdit extends React.PureComponent {
           <div className="modal-overlay" onClick={this.onModalOverlayClick} />
           <div className="modal">
             <TopSiteForm
-              label={editLabel}
-              url={editUrl}
+              label={editSite.label || editSite.hostname || ""}
+              url={editSite.url || ""}
```

Comment:
I had to add the '|| ""' because otherwise the component would keep it's old props.